sunpy/net/dataretriever/sources/tests/test_aia_synopsis.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import pytest
+from hypothesis import given
+
+import astropy.units as u
+
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+from sunpy.net._attrs import Time
+from sunpy.net.dataretriever.client import QueryResponse
+from sunpy.net.dataretriever.sources.aia_synopsis import AIASynopsisClient
+from sunpy.net.fido_factory import UnifiedResponse
+from sunpy.net.tests.strategies import time_attr
+from sunpy.time.timerange import TimeRange
+
+
```

Comment:
TODO: Add nm test for the client

Reply:
Or rather anything with units of length.

Reply:
Code did work but we had a rounding error.